statemachine/state.py

```diff
@@ -47,9 +47,12 @@ class State:
     >>> draft.to(producing)
     TransitionList([Transition(State('Draft', ...
 
-    The result is a `TransitionList`. Don't worry about this internal class. But the good
-    thing is that it implements the ``OR`` operator to combine transitions, so you can use the
-    ``|`` syntax to compound a list of transitions and assign to the same event.
+    The result is a `TransitionList`.
+    Don't worry about this internal class.
+    But the good thing is that it implements the ``OR`` operator to combine transitions,
+    so you can use the ``|`` syntax to compound a list of transitions and assign
+    to the same event.
+    You have to declare all transitions for a state in one single line.
```

Comment:
> You have to declare all transitions for a state in one single line.

**question**: Why do you have added this statement? Can you give me an example?

I think that there's not that blocks someone from writing on multiple lines.

## Example

Consider the [`Guess the number example`](https://python-statemachine.readthedocs.io/en/develop/auto_examples/guess_the_number_machine.html), someone can write the events like this and it also works:

```py
class GuessTheNumberMachine(StateMachine):

    start = State("Start", initial=True)
    low = State("Low")
    high = State("High")
    won = State("Won", final=True)
    lose = State("Lose", final=True)

    guess = lose.from_(low, high, cond="max_guesses_reached")
    guess = guess | won.from_(low, high, cond="guess_is_equal")
    guess = guess | low.from_(low, high, start, cond="guess_is_lower")
    guess = guess | high.from_(low, high, start, cond="guess_is_higher")    
```

Or eve the compact form:

```py
class GuessTheNumberMachine(StateMachine):

    start = State("Start", initial=True)
    low = State("Low")
    high = State("High")
    won = State("Won", final=True)
    lose = State("Lose", final=True)

    guess = lose.from_(low, high, cond="max_guesses_reached")
    guess |= won.from_(low, high, cond="guess_is_equal")
    guess |= low.from_(low, high, start, cond="guess_is_lower")
    guess |= high.from_(low, high, start, cond="guess_is_higher")
    
```

I personally prefer the "one-liner", but I would not say "have to".

What do you think?